### ISSUE TITLE
Training on the YT3D and DIW dataset.

### ISSUE THREAD

**Issue by leonzfa** [princeton-vl/YouTube3D#1]
I have tried to use the yt3d dataset to train a hourglass network. However, I didn’t find any decrease of the WKDR value. It is about 30% all the time. While WKDR value can decrease obviously when training on the DIW dataset. What is the problem? I used Adam solver with a learning rate of 1e-4. b_sort=True, b_oppi=False, bs=4. Thanks.

**Comment by wfchen-umich** [comment]
Hi, 
I suggest you to use the training code in this repo. We used RMSProp, lr 1e-4, bs12, b_data_aug

**Comment by leonzfa** [comment]
emmm.. It is really weird. I used RMSProp, lr 1e-4, bs8, b_data_aug, but the WKDR is still around 30%. What's more, I trained a model on DIW and achieved 20% WKDR on the DIW test set. However, if I test the trained model directly on YT3D, I got a WKDR of 70%!

**Comment by wfchen-umich** [comment]
did you read the YT3D dataset correctly? I suggest you to use the code in this repo for data loading and training.
